R/extrapolate_polymod.R

```diff
@@ -2,6 +2,7 @@
 #'
 #' @param population PARAM_DESCRIPTION
 #' @param age_breaks PARAM_DESCRIPTION, Default: c(seq(0, 75, by = 5), Inf)
+#' @param per_capita_household_size PARAM_DESCRIPTION, Default : NULL
 #' @return OUTPUT_DESCRIPTION
 #' @details DETAILS
 #' @examples
```

Comment:
Similar to above, provide more information on the arguments - can use the example of `age_breaks` that you have here in the above one as well.